posthog/async_migrations/migrations/0006_persons_and_groups_on_events_backfill.py

```diff
@@ -246,6 +247,21 @@ def _update_properties_column_compression_codec(self, query_id, codec):
                 sql=f"ALTER TABLE {settings.CLICKHOUSE_DATABASE}.{EVENTS_DATA_TABLE()} ON CLUSTER '{settings.CLICKHOUSE_CLUSTER}' MODIFY COLUMN {column} VARCHAR Codec({codec})",
             )
 
+    # TODO: Consider making postcheck a native component of the async migrations spec
+    def _postcheck(self, query_id: str, skip_on_tests=True):
+        if settings.TEST and skip_on_tests:
+            return
+        incomplete_events_ratio = sync_execute(
+            "SELECT countIf(empty(person_id) OR person_created_at = toDateTime(0)) / count() FROM events",
```

Comment:
Does the out of sync happen equally between person_created_at and properties? In other words, should we check both or this is enough